### ISSUE TITLE
How to resume training

### ISSUE THREAD

**Issue by hiteshvaidya** [lvapeab/nmt-keras#43]
Hey,

Is there a way to resume training from where it had stopped? I was training the model and it suddenly stopped due to some reasons. Now I want to resume the training, what could be the possible way?

Thanks

**Comment by hiteshvaidya** [comment]
Thanks for the update. I am getting the following error,
`NotFoundError (see above for traceback): Resource __per_step_10/_tensor_arraysstates_ta_2_8/N10tensorflow11TensorArrayE does not exist.
	 [[Node: training/Adam/gradients/decoder_AttConditionalLSTMCond/TensorArrayStack/TensorArrayGatherV3_grad/TensorArrayGrad/TensorArrayGradV3 = TensorArrayGradV3[_class=["loc:@decoder_AttConditionalLSTMCond/TensorArray_2"], source="training/Adam/gradients", _device="/job:localhost/replica:0/task:0/cpu:0"](decoder_AttConditionalLSTMCond/TensorArray_2/_2163, decoder_AttConditionalLSTMCond/while/Exit_2/_2165)]]
	 [[Node: training/Adam/gradients/bidirectional_encoder_LSTM/while/Tanh_1_grad/TanhGrad/_2955 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/gpu:0", send_device="/job:localhost/replica:0/task:0/cpu:0", send_device_incarnation=1, tensor_name="edge_13100_training/Adam/gradients/bidirectional_encoder_LSTM/while/Tanh_1_grad/TanhGrad", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"](^_clooptraining/Adam/gradients/bidirectional_encoder_LSTM/while/Select_grad/zeros_like/_1582)]]`

**Comment by lvapeab** [comment]
Dear @hiteshvaidya ,

I'm unable to reproduce your error. Are you training on a multi-gpu environment?

**Comment by hiteshvaidya** [comment]
Yes I am using multi gpu

**Comment by lvapeab** [comment]
Are you capable to load another (multigpu) models in Keras?

**Comment by hiteshvaidya** [comment]
Capable as in? If you mean resources, then yes I have sufficient resources to load another model.

**Comment by hiteshvaidya** [comment]
Ohh no problem. I haven't tried saving/reloading a model in keras before this. I will try doing that.

**Comment by lvapeab** [comment]
Hi @qazs ,

it depends on what changes did you do in the config. If you change the network architecture (model, dimensions, vocabularies), you'll cannot resume the training, because the old model (the one you load) and the new one (specified by the config), are incompatible. If the model is the same, you can reload and train with the new settings.

**Comment by qazs** [comment]
@lvapeab Got it, thanks!
